README.md

```diff
@@ -264,11 +264,11 @@ to `true`, the following outputs are available for subsequent steps that call th
 
 ## License
 
-[Mozilla Public License v2.0](https://github.com/hashicorp/setup-terraform/blob/master/LICENSE)
+[Mozilla Public License v2.0](LICENSE)
```

Comment:
No need to give an absolute URL to these documents - fixes the branch reference issue.